README.md

```diff
@@ -153,7 +153,7 @@ Key name for column in `row-data` items
 Name for column header
 
 ### type
-Data type, possible vales: `datetime`, `date`, `text`, `numeric`, `currency`, `boolean`, `percent`.
+Data type, possible vales: `datetime`, `date`, `text`, `numeric`, `currency`, `boolean`, `percent`. `select`
```

Comment:
There is a dot (instead of comma) before the last word.
Though it's only a README file, it should be fixed.